### PR TITLE
issue 5083 fix

### PR DESCRIPTION
**Pull request by PrakashDurlabhji**

### CODE REVIEW

File: src/shared/components/challenge-listing/Filters/ChallengeFilters.jsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ export default function ChallengeFilters({
   if (filterState.events && filterState.events.length) filterRulesCount += 1;
   if (filterState.tags && filterState.tags.length) filterRulesCount += 1;
   if (filterState.types && filterState.types.length) filterRulesCount += 1;
+  if (filterState.status && filterState.status !== 'Active') filterRulesCount += 1;
```
Comment (luizrrodrigues) [inline src/shared/components/challenge-listing/Filters/ChallengeFilters.jsx:52]: `status !== All`

**Review by luizrrodrigues — CHANGES_REQUESTED** [review]

**Comment by PrakashDurlabhji** [comment]
@luizrrodrigues updated

**Review by luizrrodrigues — CHANGES_REQUESTED** [review]
https://github.com/topcoder-platform/community-app/issues/5083#issuecomment-709585110

**Comment by PrakashDurlabhji** [comment]
@luizrrodrigues can you please check now? it is updated again

File: src/shared/components/challenge-listing/Filters/ChallengeFilters.jsx [hunk 2]
```diff
@@ -49,6 +50,7 @@ export default function ChallengeFilters({
   if (filterState.events && filterState.events.length) filterRulesCount += 1;
   if (filterState.tags && filterState.tags.length) filterRulesCount += 1;
   if (filterState.types && filterState.types.length) filterRulesCount += 1;
+  if (activeBucket === BUCKETS.ALL && filterState.status && filterState.status !== 'Active') filterRulesCount += 1;
```
Comment (luizrrodrigues) [inline src/shared/components/challenge-listing/Filters/ChallengeFilters.jsx:53]: Previous issue back again: https://github.com/topcoder-platform/community-app/issues/5083#issuecomment-709555852

Please test locally to make sure all working well before mark as ready.

Thanks.

**Review by luizrrodrigues — CHANGES_REQUESTED** [review]

File: src/shared/components/challenge-listing/Filters/ChallengeFilters.jsx (hunk 2 above)
Comment (PrakashDurlabhji) [reply src/shared/components/challenge-listing/Filters/ChallengeFilters.jsx]: sorry i tested a different string and pasted here different string. now its corrected

**Comment by PrakashDurlabhji** [comment]
@luizrrodrigues working fine

**Review by luizrrodrigues — APPROVED** [review]

**Pull request merged**